SimulationAgent/Agent.cs

```diff
@@ -25,6 +29,7 @@ public interface ISimulationAgent
         Task AddDeviceAsync(string simulationId, string name, string modelId);
         Task DeleteDevicesAsync(List<string> ids);
         void Stop();
+        Task SeedAsync(string templateName);
```

Comment:
I don't think having this logic in the SimulationAgent is the right move, it would be great if this agent only responsibility was simulating devices.  What about adding a new agent with this specific responsibility? Looks like many microservices will need a similar feature, so having a defined class hierarchy for this tasks would be reusable across solutions.